### ISSUE TITLE
Support for Bilibili.com seems broken

### ISSUE THREAD

**Issue by godly-devotion** [rg3/youtube-dl#4660]
The script for `bilibili.com` seems to have been last modified 25 days ago so I am not entirely sure if I am doing something wrong but trying to play the video results in an error.

I am on version `2015.01.07.2`

```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://www.bilibili.com/video/av1826069/']
[debug] Encodings: locale cp1252, fs mbcs, out None, pref cp1252
[debug] youtube-dl version 2015.01.07.2
[debug] Python version 2.7.8 - Windows-8-6.2.9200
[debug] exe versions: ffmpeg N-50911-g9efcfbe
[debug] Proxy map: {}
ERROR: Unable to extract player params; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "youtube_dl\YoutubeDL.pyo", line 588, in extract_info
  File "youtube_dl\extractor\common.pyo", line 246, in extract
  File "youtube_dl\extractor\bilibili.pyo", line 59, in _real_extract
  File "youtube_dl\extractor\common.pyo", line 502, in _html_search_regex
  File "youtube_dl\extractor\common.pyo", line 492, in _search_regex
RegexNotFoundError: Unable to extract player params; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

Example urls:
http://www.bilibili.com/video/av1826069/
http://www.bilibili.com/video/av1867709/
http://www.bilibili.com/video/av1855376/

**Comment by dstftw** [comment]
Fixed and will be available in the next release. Thanks for the report.

**Comment by godly-devotion** [comment]
Thanks for your work and quick fix.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.01.08. See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by godly-devotion** [comment]
I can confirm it has been fixed. Thanks.
